test/unit/component_snapshot_mixin.js

```diff
@@ -0,0 +1,102 @@
+import { toMatchImageSnapshot } from 'jest-image-snapshot'
+expect.extend({ toMatchImageSnapshot })
+import { getFlatUsecases } from '../../src/component_utils'
+import Vue from 'vue'
+import puppeteer from 'puppeteer-core'
+
+export default function testSnapshots (component) {
+    describe('rendered image', () => {
+        const fs = require('fs')
+        const path = require('path')
+        const compiledScript = fs.readFileSync(path.join(__dirname, '../../dist/view.js'))
+
+        let browser
+        beforeAll(async () => {
+            browser = await puppeteer.launch({
+                executablePath: '/usr/bin/chromium-browser', // Available inside Docker container
+                args: ['--disable-dev-shm-usage', '--no-sandbox', '--disable-setuid-sandbox'],
+            })
+        })
+
+        const usecases = getFlatUsecases(component)
+
+        jest.setTimeout(usecases.length * 20 * 1000)
```

Comment:
Rule of thumb? 20 secs for each usecase?

Reply:
Yes, seems to be a safe estimate in practice.

Reply:
:+1: